### PR TITLE
DOCS-9416 share vs publish

### PR DESCRIPTION
**Pull request by wwarner-salesforce**

### CODE REVIEW

File: modules/ROOT/pages/to-share-an-http-api.adoc [hunk 1]
```diff
@@ -1,17 +1,17 @@
-= Share an HTTP API Asset
+= Publish an HTTP API Asset
```
Comment (LucasLudueno) [inline modules/ROOT/pages/to-share-an-http-api.adoc]: I would also change the name of the file `to-share-an-http-api.adoc` to `to-publish-an-http-api.adoc` (and replacing it in the places it is)

**Review by LucasLudueno — DISMISSED** [review]

File: modules/ROOT/pages/to-share-an-http-api.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/to-share-an-http-api.adoc]: Good call!

**Review by LucasLudueno — DISMISSED** [review]

File: modules/ROOT/pages/publish-an-http-api.adoc [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
+:page-aliases: to-share-an-http-api.adoc
 
 The HTTP API lets Private Exchange users specify a placeholder for an endpoint. After you create the HTTP API endpoint, you can optionally manage the endpoint with API Manager.
 
-The HTTP API asset contains a description for what the API endpoint does and how users can utilize the API for use with their programs.
+The HTTP API asset contains a description of what the API endpoint does and how users can connect the API to their programs.
```
Comment (priyaayyar) [inline modules/ROOT/pages/publish-an-http-api.adoc]: The HTTP API asset explains or describes? (fewer words)

File: modules/ROOT/pages/publish-an-http-api.adoc [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
 
-Users utilize API Manager to import the API from Exchange and can then associate proxies and manage policies and SLAs for the API.
+In API Manager, users import the API from Exchange and can then associate proxies and manage policies and SLAs for the API.
```
Comment (priyaayyar) [inline modules/ROOT/pages/publish-an-http-api.adoc:12]: Exchange (is it the first instance in the topic?) If so, use Anypoint Exchange (Exchange)

File: modules/ROOT/pages/publish-an-http-api.adoc [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
 
 . In a Private Exchange, click New.
 . Give the API a name and in Asset Type, click API - HTTP.
 . Click Publish.
 . In the asset view, click Edit to provide a description.
-. Click Save As Draft and Publish.
\ No newline at end of file
+. Click Save As Draft and Publish.
```
Comment (priyaayyar) [inline modules/ROOT/pages/publish-an-http-api.adoc]: Click *Save As Draft and Publish*.

Comment (priyaayyar) [reply modules/ROOT/pages/publish-an-http-api.adoc]: In Bold

**Review by priyaayyar — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/publish-an-http-api.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/publish-an-http-api.adoc]: That is better. Fixed.

File: modules/ROOT/pages/publish-an-http-api.adoc (hunk 3 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/publish-an-http-api.adoc:12]: Updated the first instance in the file to use Anypoint Exchange.

File: modules/ROOT/pages/publish-an-http-api.adoc (hunk 4 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/publish-an-http-api.adoc]: Updated.

**Review by priyaayyar — APPROVED** [review]

**Pull request merged**